### ISSUE TITLE
inteltbb.devel.win.targets file path issues

### ISSUE THREAD

**Issue by tibi-extera** [oneapi-src/oneTBB#768]
I am using the `intelmkl.static.win-x64` package witch depends on `inteltbb.devel.win`. When building my project I got an error that `tbb12_debug.dll` or `tbb12.dll` were not found when compiling Debug and Release respectively.

I tracked the problem to the `inteltbb.devel.win.targets` file. In it, there are item group definitions for the different build options and one for the compiler. All have a similar problem. I will provide an example for the X64 debug link group.

The original group was defined as:

```xml
  <ItemDefinitionGroup Condition="$(Configuration.ToLower().Contains('debug')) AND '$(Platform)' == 'x64'">
    <Link>
      <AdditionalLibraryDirectories>$(MSBuildThisFileDirectory)..\..\lib\native\win-x64;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
      <AdditionalDependencies>tbb12_debug.lib;tbbmalloc_debug.lib;tbbmalloc_proxy_debug.lib;%(AdditionalDependencies)</AdditionalDependencies>
    </Link>
  </ItemDefinitionGroup>
```

The problem is that the location of the libraries is not `$(MSBuildThisFileDirectory)..\..\lib\native\win-x64;` but rather `$(MSBuildThisFileDirectory)..\..\..\lib\native\win-x64;`.
I have modified the file manually as follows:

```xml
  <ItemDefinitionGroup Condition="$(Configuration.ToLower().Contains('debug')) AND '$(Platform)' == 'x64'">
    <Link>
      <AdditionalLibraryDirectories>$(MSBuildThisFileDirectory)..\..\..\lib\native\win-x64;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
      <AdditionalDependencies>tbb12_debug.lib;tbbmalloc_debug.lib;tbbmalloc_proxy_debug.lib;%(AdditionalDependencies)</AdditionalDependencies>
    </Link>
  </ItemDefinitionGroup>
```

While this works for the moment on my computer, this is not a portable solution. Any other developer would have to redo the same changes I did, with the potential of introducing errors. The proper way to address this is to change the package and this is what I am asking you to do.

**Comment by anton-potapov** [comment]
@tibi-extera which version of oneTBB do you use ?
@AlexVeprev could you please take a look ?

**Comment by tibi-extera** [comment]
![image](https://user-images.githubusercontent.com/62777985/152926390-c11d6999-b76d-4fed-8832-4386afe3cdd5.png)

**Comment by AlexVeprev** [comment]
Hi @tibi-extera,
The issue looks like duplicate of #606. It was fixed, but not released with 2021.5 (see comments in #606)

**Comment by tibi-extera** [comment]
Hi @AlexVeprev, 

You are right they are the same issue. In addition #606 mentions the missing lib, which I forgot to mention.
Do you have a release date when the fix will be available?

**Comment by AlexVeprev** [comment]
@tibi-extera, unfortunately I don't. oneTBB has RC for future version available on GitHub, but final release depends on other oneAPI components. Let's keep this issue open until release with the fix is available.

**Comment by isaevil** [comment]
Seems like the problem is solved in [v2021.6.0](https://github.com/oneapi-src/oneTBB/releases/tag/v2021.6.0). If any other questions are left, feel free to reopen this issue.
